backend/reservations/views/admin.py

```diff
@@ -142,144 +144,112 @@ def guest_search(arr, low, high, x):
-
-Goes without saying, but don't forward this email.
-
-This is your password, there are many like it but this one is yours. Once you use this password on a device, RoomBaht will remember you, but only on that device.
-Copy and paste this password. Because let’s face it, no one should trust humans to make passwords:
-{otp}
-http://rooms.take3presents.com/login
-
-Good Luck, Starfighter.
-
-        """
-
-        send_mail("RoomService RoomBaht",
-                  body_text,
-                  "placement@take3presents.com",
-                  [guest_new["email"]],
-                  auth_user="placement@take3presents.com",
-                  auth_password=apppass,
-                  fail_silently=False,)
-
-
-def guest_contact_exists(guest_new, otp):
-    ''' Get the ticket numbers
-        if ticket exists copy associated room number and delete the existing entry.
-        create new entry with existing room number or new room num is none previously existed.
-    '''
-    logger.info(f"[+] Guest exists, creating ticket {guest_new['email']} ticket: {guest_new['ticket_code']}")
-    room = assign_room(guest_new["product"])
     if(room is None):
+        logging.debug("[-] Out of empty rooms")
         return
 
     guest=Guest(name=guest_new['first_name']+" "+guest_new['last_name'],
         email=guest_new['email'],
         ticket=guest_new['ticket_code'],
         jwt=otp,
         room_number=room.number)
-
     guest.save()
+
     room.guest=guest
-    logger.info(f"[+] Assigned room number: {room.number}")
     room.save()
 
-
-def guest_contact_update(guest_new, otp):
-    '''  Email exists and Ticket exists. If existing guest does not match new guest, update existing entry.'''
-    existing_ticket = Guest.objects.filter(ticket=guest_new["ticket_code"])
-    room_num = existing_ticket[0].room_number
-    if(guest_new["email"]!=existing_ticket[0].email):
-        logger.debug(f"update detected - existing: {existing_ticket}, fields: {existing_ticket[0].name},{existing_ticket[0].ticket},{existing_ticket[0].room_number}")
-        existing_ticket.update(
-                              name=guest_new['first_name']+" "+ guest_new['last_name'],
-                              email=guest_new['email'],
-                              ticket=guest_new['ticket_code'],
-                              jwt=otp,
-                              room_number = room_num)
-
-
-        logger.debug(f"updated: {existing_ticket}, fields: {existing_ticket[0].name},{existing_ticket[0].ticket},{existing_ticket[0].room_number}")
-
-    else:
-        logger.debug(f'[*] Email exists, ticket exists, room exists, everything matches entry')
+    logger.info(f"[+] Assigned room number: {room.number}")
+    if(email_onboarding):
+        time.sleep(5)
```

Comment:
Looked into [gmail limits for workspaces](https://support.google.com/a/answer/6099642?sjid=15956034020097623920-NA#messagelimits&zippy=%2Cmessage-and-posting-limits) and not seeing this limit? Maybe we should just capture the error and retry if needed....

Reply:
Ohh luv it ja plz find their spec, would b great to has faster. I recall errors when running with no limits

Reply:
Having a hard time finding a "periodic" spec. Also occurs to me that if we try sending these emails in the context of a web request, the web request will time out. Wonder if it's worth trying AES SES or just force people do manually do the account reset before they can use the thing?